provider-profile-service/src/main/java/team/proximity/provider_profile_service/payment_method/BankPaymentCreator.java

```diff
@@ -8,7 +8,7 @@
 public class BankPaymentCreator implements PaymentMethodCreator {
     @Override
     public PaymentMethod create(PaymentMethodRequest request) {
-        if (request.bankName() == null || request.accountName() == null || request.bankName().isBlank()) {
+        if (request.bankName() == null || request.accountName() == null || request.bankName().isBlank()|| request.userEmail().isBlank()) {
```

Comment:
Condition is too long to read, I suggest you extract it as method

Reply:
Okay it sounds great, thanks